app/renderer/components/styles/commonStyles.js

```diff
@@ -19,6 +19,17 @@ const styles = StyleSheet.create({
     outline: 'none',
     padding: '0.4em',
     width: '100%'
+  },
+  'flyoutDialog': {
```

Comment:
I'm going to create a tracking issue to delete https://github.com/brave/browser-laptop/blob/4fb5d5ae8202b5afffff4e3ce106a64d81fbfdb4/less/forms.less#L8-L18 in favor of these lines  :-)